test/js/samples/ssr-spread-component-props-immutably/_config.js

```diff
@@ -0,0 +1,5 @@
+export default {
```

Comment:
I was not sure if a test was suitable for this. Just tell me if it feels redundant and I will remove it.

Reply:
Good question, it's a somewhat brittle test because the resulting output is tested instead of its behavior - could we change that to something where the behavior result is tested instead?

Reply:
I agree. The `server-side-rendering` tests are testing `Svelte component => HTML output`, so I thought this was at least somewhat adding a viable test, although brittle.

I could look into adding a `runtime` test perhaps? I don't see any other SSR tests there, but I think it should be possible.

Reply:
I changed it to a `runtime` test, feels a lot better.